server/game/gameSystems/PutIntoPlaySystem.ts

```diff
@@ -71,13 +73,14 @@ export class PutIntoPlaySystem<TContext extends AbilityContext = AbilityContext>
     }
 
     protected override addPropertiesToEvent(event, card: Card, context: TContext, additionalProperties): void {
-        const { controller, overrideLocation } = this.generatePropertiesFromContext(
+        const { controller, overrideLocation, entersReady } = this.generatePropertiesFromContext(
```

Comment:
Could you do me small favor and a TODO here for "rename this class and all related classes / methods as PutUnitIntoPlay"